app/controllers/hyrax/dashboard/collections_controller.rb

```diff
@@ -264,7 +264,7 @@ def update_active_fedora_collection
 
         @collection.visibility = Hydra::AccessControls::AccessRight::VISIBILITY_TEXT_VALUE_PRIVATE unless @collection.discoverable?
         # we don't have to reindex the full graph when updating collection
-        @collection.reindex_extent = Hyrax::Adapters::NestingIndexAdapter::LIMITED_REINDEX
+        @collection.try(:reindex_extent=, Hyrax::Adapters::NestingIndexAdapter::LIMITED_REINDEX)
```

Comment:
should a similar change happen in app/jobs/reindex_collections_job.rb ?

Reply:
💯

Reply:
(or is the job obsolete now?)